pkg/scheduler/profile/profile.go

```diff
@@ -0,0 +1,38 @@
+/*
+Copyright (c) Microsoft Corporation.
+Licensed under the MIT license.
+*/
+
+// Package profile holds the definition of a scheduling Profile.
+package profile
+
+import (
+	"go.goms.io/fleet/pkg/scheduler/framework"
+	"go.goms.io/fleet/pkg/scheduler/framework/plugins/clusteraffinity"
+	"go.goms.io/fleet/pkg/scheduler/framework/plugins/clustereligibility"
+	"go.goms.io/fleet/pkg/scheduler/framework/plugins/sameplacementaffinity"
+	"go.goms.io/fleet/pkg/scheduler/framework/plugins/topologyspreadconstraints"
+)
+
+const (
+	// defaultProfileName is the default name for the scheduling profile.
+	defaultProfileName = "DefaultProfile"
+)
+
+// NewDefaultProfile creates a default scheduling profile.
+func NewDefaultProfile() *framework.Profile {
```

Comment:
where is this used?

Reply:
it will be used in the main func where it creates the scheduler, profile and framework.